gr-blocks/include/gnuradio/blocks/complex_to_interleaved_short.h

```diff
@@ -39,7 +39,7 @@ class BLOCKS_API complex_to_interleaved_short : virtual public sync_interpolator
     /*!
      * Build a complex to interleaved shorts block.
      */
-    static sptr make(bool vector = false);
+    static sptr make(float scale_factor = 1.0, bool vector = false);
```

Comment:
Since `scale_factor` is a potentially runtime-settable parameter, please add in the methods for doing so.